cmd/kube-apiserver/app/options/options.go

```diff
@@ -155,9 +155,11 @@ func (s *ServerRunOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.StringVar(&s.SSHUser, "ssh-user", s.SSHUser,
 		"If non-empty, use secure SSH proxy to the nodes, using this user name")
+	fs.MarkDeprecated("ssh-user", "This flag will be removed in a future version.")
```

Comment:
add a comment to track when it was deprecated (e.g. "// deprecated in 1.9")

Reply:
Done!